### ISSUE TITLE
Typo in shortcode-chart-js-plusplus.yaml

### ISSUE THREAD

**Issue by fugo** [sal0max/grav-plugin-shortcode-chart-js-plusplus#2]
there are to much numbers in the hex code color at the default plugin configuration:
``#65a0e999`` 

```
enabled: true
chart:
  backgroundColor: '#65a0e999' 
  borderColor: '#65a0e9'
  pointBackgroundColor: '#65a0e9'
  pointBorderColor: '#65a0e9'
```

**Comment by sal0max** [comment]
#65a0e999 is still a valid hex color code. In this case, 65 is the alpha value.

I don't know where I got that from. Probably somewhere from the chart.js documentation.
Don't know if it's a typo or intended. Without Alpha probably makes more sense. So yeah, I'll change it.

Thanks for your message.

**Comment by fugo** [comment]
If you want transparency it is more easy with RGBA
With Hex its a bit comlicated
The right code Hex would be **66** for 40%, **63** for 39% or **69** for 41% :smirk: 

[Here you find a overview - gist.github.com ](https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4)

**Comment by sal0max** [comment]
Thanks for the explanation. 👍

Fixed in 14ddd5244e23dc4bc0aacdcada61e023069f95aa
